### DISCUSSION TITLE
Allow Multiple Private Repositories in a Sponsor Tier on GitHub

### DISCUSSION THREAD

**Discussion by EduApps-CDG** [community/community#51275] [Other Feature Feedback, Questions, & Ideas]
GitHub Sponsors is a great platform. However, I find it limiting that I can only support one private repository per tier. Many developers, myself included, have multiple private repositories that we would like to offer to our sponsors.

It would be great if GitHub Sponsors could allow sponsors to support multiple private repositories in a single tier. This would not only provide more flexibility for sponsors to choose which repositories they want to support, but it would also benefit developers by allowing them to offer more value to their sponsors. I believe that this feature would enhance the overall experience of GitHub Sponsors and encourage more sponsors to support their favorite open-source projects.

For example, think in a user that has a project splitted in 5 repositories (desktop project, plugins, android app, etc) and he wants to allow its sponsors to have access to all of these repos as part of a bigger project. In this scenario, the current limitation of only allowing one private repository per tier makes it difficult for the user to offer their sponsors access to all of their repositories. This can be a major drawback for the user, as it limits the potential support they can receive from their sponsors.

I would greatly appreciate if you could consider implementing this feature and look forward to seeing it added to the platform. Thank you for your time and consideration.

----

**TLDR:** Allow sponsors to support multiple private repositories in a single tier.

**Comment by LlmDl** [comment]
Duplicate of https://github.com/orgs/community/discussions/17986

**Reply by EduApps-CDG** [reply]
lol
